### DISCUSSION TITLE
BROADCAST multiple platforms at the same time natively

### DISCUSSION THREAD

**Discussion by HCOSCARELLI** [obsproject/obs-studio#7186] [Development Questions]
Please put a feature to stream on multiple social networks at the same time this will help a lot of people.

**Comment by derrod** [comment]
For feature requests, see our ideas page. This specific one is already one of the highest rated ones, and will follow once we make the necessary changes to the UI/UX: https://ideas.obsproject.com/posts/7/multi-service-streaming-output
